src/components/Details/TranscriptFull/TranscriptFull.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, RefObject } from "react";
+
+import TranscriptItem, { TranscriptItemRef } from "../TranscriptItem/TranscriptItem";
+
+import { Sentence } from "../../Shared/Types/Transcript";
+import hhmmss from "../../../utils/hhmmss";
```

Comment:
I'm not a huge fan of functions that don't state their purpose as part of their name. Can we name this to clearly indicate what it does?

Reply:
I can change it convertSecondsToHHMMSSString. Is there a better variable name for hhmmss part?

Reply:
Yeah I'll give a +1 for using an overly descriptive function/variable naming.

Also I think `convertSecondsToHHMMSSString` is solid.

You could also use `secondsToHHMMSS` as a compromise between length and while still conveying what the function does.

Reply:
Oh also I think HHMMSS is fine. You could be more vague and say something like TimeString, but in this case I think it's helpful to be specific about what type/format of time string you're returning.